charts/trillian/README.md

```diff
@@ -47,21 +47,19 @@ helm uninstall [RELEASE_NAME]
 | createdb.image.repository | string | `"sigstore/scaffolding/createdb"` |  |
 | createdb.image.version | string | `"sha256:9aa98492115c465b0cecfd6dbb04411a40c0d2d7e5d7c510f5646bd1d825e3c7"` | v0.6.2 |
 | createdb.name | string | `"createdb"` |  |
-| createdb.resources | string | `""` |  |
```

Comment:
I don't think we want to delete this fields.

Reply:
I'll try to update this this week... I'm not sure how that got deleted because I think I just used `helm-docs --dry-run` to generate it.

Reply:
I checked and this didn't have a default in values.yaml, so I added it as the correct `map` type.